### PR TITLE
ADS: Slider specs

### PR DESCRIPTION
**Pull request by nalcalag**
Task/Issue URL: https://app.asana.com/0/1202857801505092/1203433470862092/f

### Description
Update track and thumb colors for Slider

### Steps to test this PR
- Install from this branch
- Go to Settings > Design Preview
- Tap on Interactive tab
- [ ] Check Slider colors match designs

### UI changes
| Light  | Dark |
| ------ | ----- |
Screenshot_20221124-172436_DuckDuckGo|Screenshot_20221124-172446_DuckDuckGo|

### CODE REVIEW

**Comment by nalcalag** [comment]
Current dependencies on/for this PR:
* develop
  * **PR #2561** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/2561" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/duckduckgo/Android/2561?utm_source=stack-comment).

File: common-ui/src/main/res/values/design-system-theming.xml [hunk 1]
```diff
@@ -277,6 +277,11 @@
         <item name="daxColorSwitchThumbOn">?attr/daxColorAccentBlue</item>
         <item name="daxColorSwitchThumbOff">@color/gray40</item>
 
+        <!-- Slider -->
+        <item name="daxColorSliderTrackInactive">@color/gray40_50</item>
+        <item name="daxColorSliderTrackActive">?attr/daxColorAccentBlue</item>
```
Comment (malmstein) [inline common-ui/src/main/res/values/design-system-theming.xml:282]: `daxColorSliderTrackActive` and `daxColorSliderTrackActive` have the same values in both themes, should we just have one `daxColorSliderActive`?

File: common-ui/src/main/res/values/widgets.xml [hunk 2]
```diff
@@ -371,17 +371,10 @@
 
     <!-- Slider -->
     <style name="Widget.DuckDuckGo.Slider" parent="Widget.MaterialComponents.Slider">
-        <item name="materialThemeOverlay">@style/ThemeOverlay.DuckDuckGo.Slider</item>
-        <item name="trackHeight">6dp</item>
-        <item name="trackColorInactive">@color/slider_inactive_track</item>
```
Comment (malmstein) [inline common-ui/src/main/res/values/widgets.xml:376]: if we are not using this color anymore, can we remove it?

**Review by malmstein — APPROVED** [review]
couple nits, looking good!

**Pull request merged**